utils/zerovec/src/ule/multi.rs

```diff
@@ -44,7 +44,7 @@ impl MultiFieldsULE {
                 lengths, output,
             );
             debug_assert!(
-                <VarZeroSlice<[u8]>>::validate_byte_slice(output).is_ok(),
+                <VarZeroSlice<[u8], Index32>>::validate_byte_slice(output).is_ok(),
```

Comment:
Good sleuthing!

However, I think we should prefer Index16. No reason to use the extra bytes.

Reply:
cc @Manishearth

Reply:
Yeah, this is intended to default to Index16. In the long run we can introduce a derive config flag for this.

Also it's a data breaking change to change the format here, though I don't think ICU4X uses this yet (it will once casemap stabilizes)

Reply:
wait, no, nvm, it should default to 32 since we expect the lengths to be large

Reply:
That's the issue of this PR, it is Index32 but not consistently. Switching to Index16 would be a breaking change as I understand

Reply:
(VarZeroVec defaults to Index16, but MFULE overrides that to Index32)